Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'capistrano-s3'
+gem 'dotenv-rails'
```

Comment:
I guess all this is still in flux, so maybe these questions don't need to be answered now.